dev/com.ibm.ws.kernel.instrument.agent/src/com/ibm/ws/kernel/instrument/BootstrapAgent.java

```diff
@@ -1,5 +1,5 @@
 /*******************************************************************************
- * Copyright (c) 2010, 2014 IBM Corporation and others.
+ * Copyright (c) 2010, 2014, 2018 IBM Corporation and others.
```

Comment:
Only ever have two dates here, the first and last ... so this would be `2010, 2018`

Reply:
Thank you for your comment. I'll fix it to the next PR which update this function.